skip broken test, see https://github.com/numpy/numpy/issues/16769
author: Kenneth Hoste (HPC-UGent)
updated for version 1.22.3 by Jasper Grimm (UoY)
-----------------------------------------------------------------
diff -Nru numpy-1.22.3.orig/numpy/distutils/tests/test_system_info.py numpy-1.22.3/numpy/distutils/tests/test_system_info.py
--- numpy-1.22.3.orig/numpy/distutils/tests/test_system_info.py	2022-05-19 13:52:55.000000000 +0100
+++ numpy-1.22.3/numpy/distutils/tests/test_system_info.py	2022-05-19 13:54:24.000000000 +0100
@@ -258,7 +258,7 @@
 
     @pytest.mark.xfail(HAS_MKL, reason=("`[DEFAULT]` override doesn't work if "
                                         "numpy is built with MKL support"))
-    def test_overrides(self):
+    def ignored_test_overrides(self):
         previousDir = os.getcwd()
         cfg = os.path.join(self._dir1, 'site.cfg')
         shutil.copy(self._sitecfg, cfg)
